api/test/unit/services/listing.service.spec.ts

```diff
@@ -2391,6 +2399,15 @@ describe('Testing listing service', () => {
         id: 'example id',
         name: 'example name',
       });
+      prisma.listingEvents.findMany = jest.fn().mockResolvedValue([]);
+      prisma.listingEvents.update = jest.fn().mockResolvedValue({
+        id: 'example id',
+        name: 'example name',
+      });
+      prisma.assets.delete = jest.fn().mockResolvedValue({
```

Comment:
Could we add a test to this PR? Maybe just expecting this to be called?

Reply:
I ended up writing a separate set of tests for this function, lemme know if that is okay or if you would prefer the tests being part of this